tests/unit/models/cart-model-test.js

```diff
@@ -350,3 +350,34 @@ test('it keeps line_item attrs hashes, and doesn\'t inject classes into internal
     done();
   });
 });
+
+
+test('it updates line_item attrs correctly when removing a line item', function (assert) {
```

Comment:
What is `correctly`?

Specifically: `it doesn't pollute "attrs.line_items" with "CartLineItem" class instances on #removeLineItem`